### DISCUSSION TITLE
deye_2mppt.yaml is missing 1.4.3

### DISCUSSION THREAD

**Discussion by derAllgaeu** [StephanJoubert/home_assistant_solarman#351] [Q&A]
Hi!

I downloaded the integration from HACS and want to configure my SUN-M80G3-EU-Q0-800W
For that i want to use the recommended deye_2mppt.yaml but it is missing in the Integration configure dialog.
I only can choose the deye_4mppt.yaml

What can I do? How can I implement the deye_2mppt.yaml afterwards?

Kind regards,
derQ

**Comment by derAllgaeu** [comment]
Now I downloaded the deye_2mppt.yaml and added it manually to the share where all definitions are on the filesystem.
Then I rebootet the HA system, but I still can't choose it in the cofigure dialog.

Can you give me a hint?

**Comment by Harlekien** [comment]
Hello derAllgaeu,

I had the same issue. You have to add the "deye_2mppt.yaml" in the "const.py" file. The "const.py" file is located in the solarman folder. After that, you have to restart HA. Now you should be able to choose the "deye_2mppt.yaml" in the configuration for MC.

This is my "const.py" 
```
from datetime import timedelta

DOMAIN = 'solarman'

DEFAULT_PORT_INVERTER = 8899
DEFAULT_INVERTER_MB_SLAVEID = 1
DEFAULT_LOOKUP_FILE = 'deye_hybrid.yaml'
LOOKUP_FILES = [
    'deye_2mppt.yaml',
    'deye_4mppt.yaml',
    'deye_hybrid.yaml',
    'deye_sg04lp3.yaml',
    'deye_string.yaml',
    'sofar_lsw3.yaml',
    'sofar_wifikit.yaml',
    'solis_hybrid.yaml',
    'solis_1p8k-5g.yaml',
    'sofar_g3hyd.yaml',
    'sofar_hyd3k-6k-es.yaml',
    'zcs_azzurro-ktl-v3.yaml',
    'custom_parameters.yaml'
]

MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=30)

CONF_INVERTER_HOST = 'inverter_host'
CONF_INVERTER_PORT = 'inverter_port'
CONF_INVERTER_SERIAL = 'inverter_serial'
CONF_INVERTER_MB_SLAVEID = 'inverter_mb_slaveid'
CONF_LOOKUP_FILE = 'lookup_file'

SENSOR_PREFIX = 'Solarman'
```
